staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2413,6 +2416,20 @@ type KubeVirtConfiguration struct {
 	AutoCPULimitNamespaceLabelSelector *metav1.LabelSelector `json:"autoCPULimitNamespaceLabelSelector,omitempty"`
 	// LiveUpdateConfiguration holds defaults for live update features
 	LiveUpdateConfiguration *LiveUpdateConfiguration `json:"liveUpdateConfiguration,omitempty"`
+
+	// VmRolloutStrategy defines how changes to a VM object propapate to its VMI
+	VmRolloutStrategy *VmRolloutStrategy `json:"vmRolloutStrategy,omitempty"`
+}
+
+type VmRolloutStrategy struct {
+	LiveUpdate *RolloutStrategyLiveUpdate `json:"liveUpdate,omitempty"`
+	Stage      *RolloutStrategyStage      `json:"stage,omitempty"`
+}
+
+type RolloutStrategyLiveUpdate struct {
+}
+
+type RolloutStrategyStage struct {
```

Comment:
Thanks for the explanation! In this case, it totally makes sense. :+1:

Reply:
I wish these offline review was recorded here. I cannot imagine how we would like to have two versions of (say) `VmRolloutStrategyStage` instead of adding another named strategy.